app/support/orders/item_adder.rb

```diff
@@ -1,5 +1,8 @@
 class Orders::ItemAdder
 
+  # time based services default to 1 minute (arbitrary)
+  DEFAULT_DURATION = 1
```

Comment:
Should we give a more specific name for this constant? `DEFAULT_TIMED_SERVICE_DURATION` or something similar? Mainly because it doesn't apply to instruments, but you could expect it to.